src/main/java/com/revature/g2g/services/jda/listeners/GuildVoiceEventListener.java

```diff
@@ -91,7 +91,6 @@ private void processMoveArrive(GuildVoiceUpdateEvent event) {
 		}
 	}
 	private void processMoveLeave(GuildVoiceUpdateEvent event){
-		//TODO add logic to mark person as having left.
```

Comment:
I don't see new code resolving this todo. If it is not solved, please leave the to do for a future sprint. (this specific todo is dependant on OAuth working so it is currently blocked)